openapi/components/schemas/ExternalServiceSettings.yaml

```
@@ -0,0 +1,10 @@
+type: object
+description: Settings for external service integration.
+properties:
+  quickBooksOnline:
+    type: object
+    description: QuickBooks Online settings.
+    properties:
+      syncJournalEntries:
+        type: boolean
+        description: Enable or disable journal entries synchronization.
```

```suggestion
        description: Specifies if journal entries should be synchronized.
```
https://github.com/Rebilly/api-definitions/blob/7b8746ac0d49f1c59946a9abd695c5ad22ebc513/WRITING-STYLE.md?plain=1#L255